packages/pyright-internal/src/common/pathConsts.ts

```diff
@@ -8,5 +8,7 @@
 
 export const typeshedFallback = 'typeshed-fallback';
 export const lib = 'lib';
+export const lib64 = 'lib64';
+export const Lib = 'Lib';
```

Comment:
Technically I think we got this one for free; `Lib` in caps is only on Windows, which is case insensitive anyway. Doesn't hurt.

Reply:
Thanks for the review. Are you suggesting to remove it? I wondered which platform the capitalized Lib was for.

Reply:
No, it's fine; I can see that the original code checked it (missed it on my first read).